File: event-service/src/main/java/com/nurdorproject/event_service/dto/EndEventResultDto.java
package com.nurdorproject.event_service.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class EndEventResultDto {

    private String message;
    private Long totalDonations;
    private List<StandDto> stands;
}

File: mobile-app/app/src/main/java/com/example/nurdor_volunteer_app_v3/dto/eventDto/EndEventResultDto.kt
package com.example.nurdor_volunteer_app_v3.dto.eventDto

import com.example.nurdor_volunteer_app_v3.dto.standDto.StandDto

data class EndEventResultDto(
    val message: String,
    val totalDonations: Long,
    val stands: List<StandDto>
)